typings/externs.d.ts

```diff
@@ -36,11 +36,32 @@ declare global {
       disableDeviceEmulation?: boolean;
       disableCpuThrottling?: boolean;
       disableNetworkThrottling?: boolean;
+      onlyAudits?: string[];
+      onlyCategories?: string[];
+      skipAudits?: string[];
     }
 
     // TODO: type checking for Config
     export interface Config {
       passes: ConfigPass[];
+      settings: ConfigSettings;
+    }
+
+    export interface ConfigSettings {
```

Comment:
technically doesn't this extend `Flags` ?

Reply:
yeah we'll need to decide what to do here, flags is currently what comes back fresh off yargs, which is slightly different than the flags that get passed into core which is what `ConfigSettings` extends, I'll extract the shared stuff and have both extend that 👍